### PR TITLE
update to latest Redis version

### PR DESCRIPTION
**Pull request by jdmcd**
Looking for feedback on how to implement `isConnected` - right now it's just returning `true`

### CODE REVIEW

File: Sources/QueuesRedisDriver/JobsRedisDriver.swift [hunk 1]
```diff
@@ -62,7 +62,11 @@ struct _QueuesRedisQueue {
     let context: QueueContext
 }
 
-extension _QueuesRedisQueue: RedisClient {
+extension _QueuesRedisQueue: RedisClient {    
+    var isConnected: Bool {
+        return true
```
Comment (Mordil) [inline Sources/QueuesRedisDriver/JobsRedisDriver.swift:67]: I think for now, you can just forward the `self.client.isConnected`, even though it too is just returning `true` statically.

**Review by Mordil — APPROVED** [review]

**Comment by Mordil** [comment]
I apologize for this. See https://github.com/vapor/redis-kit/pull/31#discussion_r391780672 for more context

**Review by tanner0101 — APPROVED** [review]

**Comment by Andrewangeta** [comment]
@mcdappdev 👀Approved

**Comment by tanner0101** [comment]
These changes are now available in [1.0.0-rc.2](https://github.com/vapor/queues-redis-driver/releases/tag/1.0.0-rc.2)

**Pull request merged**